src/libraries/System.Private.CoreLib/src/System/Buffers/TlsOverPerCoreLockedStacksArrayPool.cs

```diff
@@ -459,13 +459,18 @@ public void Trim(int currentMilliseconds, int id, Utilities.MemoryPressure press
                             {
                                 trimCount++;
                             }
-                            if (Unsafe.SizeOf<T>() > StackModerateTypeSize)
+                            unsafe
                             {
-                                trimCount++;
-                            }
-                            if (Unsafe.SizeOf<T>() > StackLargeTypeSize)
-                            {
-                                trimCount++;
+#pragma warning disable 8500 // sizeof of managed types
+                                if (sizeof(T) > StackModerateTypeSize)
+                                {
+                                    trimCount++;
+                                }
+                                if (sizeof(T) > StackLargeTypeSize)
+                                {
+                                    trimCount++;
+                                }
```

Comment:
Seems like this now has a single indentation level too much.

Reply:
I have added `unsafe` scope that needs an extra indentation. The function is long and the `unsafe` scope makes it easier to see what needs unsafe.

Reply:
Ah, I've missed the `unsafe` above.